components/core/src/clp_s/ArchiveWriter.cpp

```diff
@@ -33,10 +33,10 @@ void ArchiveWriter::open(ArchiveWriterOption const& option) {
     m_archive_path = archive_path.string();
     if (false == std::filesystem::create_directory(m_archive_path, ec)) {
         SPDLOG_ERROR(
-                "Failed to create archive directory \"{}\" - ({}) {}",
+                "Failed to create archive directory \"{}\" - {} ({})",
```

Comment:
Can you undo this line? I only meant we should change the order of the args. So the output should be (error value) error message.